src/call.rs

```diff
@@ -93,6 +93,12 @@ mod impls {
         }
     }
 
+    impl Convert<*mut libc::c_char> for CString {
+        fn convert(&self) -> *mut libc::c_char {
+            self.as_ptr() as *mut libc::c_char
+        }
+    }
+
```

Comment:
The pointer returned by `as_ptr` should never be written to. See https://doc.rust-lang.org/std/ffi/struct.CString.html#method.as_ptr